src/main/scala/org/ergoplatform/Emission.scala

```diff
@@ -0,0 +1,85 @@
+package org.ergoplatform
+
+object Emission {
+
+  val CoinsInOneErgo: Long = 1000000000
+
+  /**
+    * Emission rules.
+    * Return number of coins, issued at height `h` and all previous heights
+    */
+  def issuedCoinsAfterHeight(h: Int,
```

Comment:
@catena2w consider renaming to `issuedCoinsAtHeight`, this is also consistent with the next method.